### DISCUSSION TITLE
Add Command Line installation

### DISCUSSION THREAD

**Discussion by jeremy-bepoix** [ggarra13/mrv2#275] [Ideas]
Adding a few parameters would make installation easier.

**Comment by ggarra13** [comment]
Hi, Jeremy.  Linux and macOS installation unfortunately don't allow command-line parameters.  Windows has support for the /Q (Quiet) flag I believe, but some options like file associations still need to bring up the UI.
The best solution if you are using mrv2 on a network is to install it on a central location.  For global configuration files, you can use %STUDIOPATH% ($STUDIOPATH on Linux and macOS).

In any case, if you can open an enhancement Issue and provide more details (OS, etc) on how you would like it to work it would be useful.

**Reply by jeremy-bepoix** [reply]
Sorry for the delay, As you recommend, I'm going to do a zip installation.
